epam-electric/components/widgets/StatusIndicator.tsx

```diff
@@ -0,0 +1,14 @@
+import { createSkinComponent } from '@epam/uui-core';
+import { StatusIndicator as UuiStatusIndicator, StatusIndicatorColors, StatusIndicatorCoreProps } from '@epam/uui';
+
+export type StatusIndicatorMods = {
+    /** @default 'gray' */
+    color?: StatusIndicatorColors | 'white' | 'gray' | 'yellow'| 'orange' | 'fuchsia' | 'purple' | 'violet' | 'cobalt' | 'cyan' | 'mint';
+};
+
+export type StatusIndicatorProps = StatusIndicatorCoreProps & StatusIndicatorMods;
+
+export const StatusIndicator = createSkinComponent<StatusIndicatorCoreProps, StatusIndicatorProps>(
```

Comment:
The same issue with first generic as for Badge

Reply:
fixed